test/workers/delete_object_hierarchy_worker_test.rb

```diff
@@ -6,6 +6,8 @@ class DeleteObjectHierarchyWorkerTest < ActiveSupport::TestCase
   include ActiveJob::TestHelper
 
   setup do
+    next if self.class != DeleteObjectHierarchyWorkerTest
```

Comment:
makes all tests inheriting from this one faster because a useless metric is not being created every time.